nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/trace/Tracer.java

```diff
@@ -64,7 +67,8 @@ protected void pushContext(OperationType chain) {
 
     protected void popContext() {
         Context context = stack.pop();
-        Trace trace = factory.newTrace(context.parent, context.typeof, context.calls, context.calls.getLast().getOutput(), context.error);
+        Trace trace = factory.newTrace(context.parent, context.typeof, context.calls,
+                context.calls.getLast().getOutput(), context.error);
```

Comment:
There's no actual content changes in this file, besides reformatting. You should leave it out of your commit.